docs/DeveloperGuide.md

```diff
@@ -388,7 +390,51 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 3d2. List is not empty.
     
         ConTActs will modify all tutorials for every Person.
-         
+
+**UC05: Edit student details**
+
+**MSS**
+
+1.  User requests to list persons.
```

Comment:
Should we change persons here to students to standardise with other UCs and the rest of the DG?

#209 dependent

Reply:
+1